src/controllers/cartClear.controllers.js

```diff
@@ -0,0 +1,10 @@
+import { cartServices } from '../services';
+import { asyncWrapper } from '../helpers';
+
+const clearCartItems = asyncWrapper(async (req, res) => {
+  const userId = req.user.id;
+  const data = await cartServices.clearCart(userId);
+  return res.status(200).json({ code: 200, message: 'Cart cleared', data });
+});
```

Comment:
You can move `asyncwrapper()` into routes. for code readability

Reply:
this function is totally reusable, as it has to have a try-catch block. it is independent and can be used wherever.

Reply:
done too